test/common/modules_loader.js

```diff
@@ -297,10 +297,9 @@ var modulesLoader = new function() {
 			this.logger,
 			(err, __cache) => {
 				if (err) {
-					cb(err, __cache);
-				} else {
-					this.initModule(Cache, _.merge(this.scope, { cache: __cache }), cb);
+					return cb(err, __cache);
 				}
+				this.initModule(Cache, _.merge(this.scope, { cache: __cache }), cb);
```

Comment:
no return here?

Reply:
Current lint doesn't catch this. However, it will be caught by another rule that I'll introduce later `consistent-return`.

Reply:
ok, that should be fine.